### PR TITLE
protobuf 3.15.* & version/build bump

### PR DESCRIPTION
**Pull request by dwsutherland**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

Corresponding change:
https://github.com/cylc/cylc-flow/pull/4123

### CODE REVIEW

**Review by kinow — APPROVED** [review]

**Review by hjoliver — APPROVED** [review]

**Comment by dwsutherland** [comment]
Note: This PR hasn't updated the other dependencies (that may need changed for beta release).

**Review by oliver-sanders — APPROVED** [review]
Yes I think other deps will need to be changed too.

Ok with the proto change.

I think we need to hold off merging this because of the version change?

**Comment by kinow** [comment]
> Yes I think other deps will need to be changed too.
> 
> Ok with the proto change.
> 
> I think we need to hold off merging this because of the version change?

In part I think. If this PR is merged, it would use the new protobuf, but install Cylc Flow from commit `974480e2cee98071470ce8ca13eab600a32f213310b9fdc14a0b15ab6d50cb07` (in recipe). Not sure if that would work.

Besides, I think the `cylc-feedstock` wouldn't pick it. So users would have to manually install this build if they really wanted to use it (we had issues releasing last version, and locked on the build number https://github.com/conda-forge/cylc-feedstock/blob/c2e07f6e06fc4c19170959fb5053f54a12330508/recipe/meta.yaml#L24

**Comment by hjoliver** [comment]
Closing; cherry-picked to #15

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ requirements:
     - graphene >=2.1,<3
     - jinja2 ==2.11.0,<2.12
     - metomi-isodatetime ==1!2.0.1
-    - protobuf ==3.12.1
+    - protobuf =>3.15.0,<3.16.0
```
Comment (hjoliver) [inline recipe/meta.yaml:29]:
```suggestion
    - protobuf >=3.15.0,<3.16.0
```

Oops, and we all approved that :rofl:

**Pull request closed without merging**